### DISCUSSION TITLE
With Heroku free tier gone has anyone [May 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5748] [Discord Threads]
> 🙋‍♂️ **jstratm**

**Comment by o1lab** [comment]
> 🙋‍♂️ **jstratm**  
#### Oracle's Always Free looks promising in theory, but likely a huge pain to manage.  
#  
> 🙋‍♂️ **spaceb0t**  
#### yes, https://neon.tech  
#  
> 🙋‍♂️ **spaceb0t**  
#### neon.tech for the DB  
#  
> 🙋‍♂️ **spaceb0t**  
#### For the app itself. I made a template for a free Dockerized web panel called https://easypanel.io  
#  
> 🙋‍♂️ **spaceb0t**  
#### Oracle free tier is not ideal for production, since they can revoke your server at any time   
####    
#### Better to pay $6 for a VPS and 1-click install EasyPanel on Vultr, Digital Ocean, Linode (or AWS)  
#  
> 🙋‍♂️ **spaceb0t**  
#### Hmm…looks like Hetzner is $4 / mo as well  
#  
> 🙋‍♂️ **jstratm**  
#### Thanks, will take a look! I also misread Railway's model- it does look free if resources used are under $5, so will be looking into that too. Also stumbled on a great resource for fly.io: https://kevinquinn.fun/blog/how-to-self-host-nocodb-on-fly.io/   
#### Guess I should have done more research : )  
#  
> 🙋‍♂️ **jstratm**  
#### But thankfully for my use case, it's not for production (yet), just exploration... and didn't want to rack up a bill in case this project takes months to gain momentum. Eventually, if NocoDB looks like the right way forward, will go for something paid.  
#  
> 🙋‍♂️ **jstratm**  
#### Had explored Grist with early success, but self hosting is more painful than NocoDB, ultimately too abstracted from the database itself, and can't handle large datasets well.  
#  
> 🙋‍♂️ **spaceb0t**  
#### Despite more effort than I would would like to admit, I have struggled to keep a grist container up and stable    
####    
#### As for railway, I think it’s a great concept, but I have never been able to make it a whole month on free tier resources. That isn’t to say it’s not worth paying for, it’s just my own experience  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1097951449595510864)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
